docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+---
+title: React Native
+---
+
+import { ImageWrapper } from "./ImageWrapper";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import JellyfishArchitecture from "./common/_jellyfish-architecture.mdx";
+import StartingJellyfishBackend from "./common/_starting-jellyfish-backend.mdx";
+import StartingJellyfishDashboard from "./common/_starting-dashboard.mdx";
+
+# React Native Minimal Working Example
+
+## What you'll learn
+
+This tutorial will guide you through creating your first React Native / Expo
+project which uses Jellyfish client. By the end of the tutorial, you'll have a
+working application that connects to the front-end dashboard using WebRTC
+technology and streams and receives camera tracks.
+
+<ImageWrapper url={'/img/tutorials/rn_full_app.gif'}/>
+
+You can check out the finished project [here](https://github.com/jellyfish-dev/jellyfish-clients-tutorials/tree/main/JellyfishDashboardRN).
+
+## What do you need
+
+- a little bit of experience in creating apps with React Native and/or Expo -
+  refer to the [React Native
+  Guide](https://reactnative.dev/docs/getting-started) or [Expo
+  Guide](https://docs.expo.dev/) to learn more
+
+## Jellyfish architecture
+
+<JellyfishArchitecture />
+
+### Start the Jellyfish backend
```

Comment:
`Starting JF backend` and `Starting JF dashboard` doesn't fit into  `Jellyfish architecture`, does it? I would suggest:

1. What you'll learn?
2. What do you need?
3. Jellyfish archutecture (?)
4. Setup 
  a) Start JF
  b) Start JF Dashboard
  c) Create React Native/Expo project
  d) Add dependencies
  e) Permissions configuration
  d) Add components library (maybe should be in `Add dependencies`)?
5. Screens
6. ConnectScreen
7. RoomScreen
 
  
  
Also, avoid using the term `Jellyfish backend` in the context of `Jellyfish Media Server.` `Jellyfish backend` is the `Jellyfish Dashboard` in this tutorial.